### DISCUSSION TITLE
테스터를 다수 등록했는데 테스터끼리 같은 방으로 모이지 않는것같습니다.

### DISCUSSION THREAD

**Discussion by deanvilgel** [naverz/zepeto-studio-kor#263] [5. Q&A]
테스터 끼리 안만나고 따로 방이 파지는 듯 한데, 해결방법이 있을까요?
인원수 제한은 16명으로 해둔 상태입니다.

**Comment by moondory77-naverz** [comment]
안녕하세요. 
현재 Pretest 상태의 월드패키지에 대해, 
(말씀주시는 환경의 테스트를 위한) 초대기능을 따로 제공드리고 있지 않습니다.

다음 제페토 앱 업데이트(3.9.7) 일정에 맞춰, 해당 기능을 지원드릴 예정이니 참고 부탁드리겠습니다.
감사합니다.
